lib/galaxy/dependencies/__init__.py

```diff
@@ -207,7 +207,7 @@ def check_total_perspective_vortex(self):
     def check_pbs_python(self):
         return "galaxy.jobs.runners.pbs:PBSJobRunner" in self.job_runners
 
-    def check_pykube_ng(self):
+    def check_pykube(self):
```

Comment:
I think it's wrong to merge https://github.com/galaxyproject/galaxy/pull/17323 forward to dev, since the reverted changes were merged again in dev via https://github.com/galaxyproject/galaxy/pull/17324 .

Reply:
@jdavcs It was easier to do it myself, so I just pushed to your branch, hope that's OK?
FYI, I've added a commit reverting 61f3db22e4da80c7d38169491ed7a6cd6cb7d8e6, then merged everything in the original merge commit to make it cleaner.

Reply:
Thank you for handling it! Absolutely OK and much appreciated!